.github/workflows/grandpa-cw-docker-image.yml

```diff
@@ -0,0 +1,74 @@
+name: "Build and publish Grandpa Docker image"
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v*'
```

Comment:
We need to first understand what is a purpose of this docker. Than we will be able to define when to build and publish this.

@blasrodri please do let me know when this image should be ready? At which stage of software development cycle?
- if there is any branch ? build and publish every commit?
- if there is any branch but with PR i.e. into master ?
- or maybe this is enough just to build and publish after merge into master branch ?
- or maybe this is enough to build and pubblich when building new release ? triggered by tag

Reply:
Let's just build only with tags that contain the cosmos word. It's for the ibc-go developers right now